var/spack/repos/builtin/packages/moab/package.py

```diff
@@ -58,7 +59,8 @@ class Moab(Package):
     depends_on('hdf5+mpi')
     depends_on('netcdf+mpi', when='+netcdf')
     depends_on('parmetis')
-    depends_on('trilinos')  # looking for zoltan.
+    depends_on('zoltan')
```

Comment:
it may as well not be needed here, just check that both
`spack spec moab` and `spac spec moab+fortran` work.